profilehooks.py

```diff
@@ -703,15 +707,25 @@ def somefunc(x, y):
 
         @timecall(timer=time.clock)
 
+    You can also log the output to a logger by specifying the name and level
+    of the logger to use, eg:
+
+        @timecall(immediate=True, log_name='profile_log', log_level=logging.DEBUG)
```

Comment:
Please keep the blank line just before the `"""` in this case.

I feel that indented sections in docstrings require padding on all three sides (top, left, bottom).